aks/application/variables.tf

```diff
@@ -102,3 +102,20 @@ variable "github_personal_access_token" {
   default     = null
   description = "Github Personal Access Token (PAT) of github_username"
 }
+
+variable "azure_resource_prefix" {
+  type        = string
+  default     = null
+  description = "Prefix of Azure resources for the service"
+}
+
+variable "service_short" {
+  type        = string
+  default     = null
+  description = "Short name of the service"
+}
```

Comment:
I'd suggest organising the variables in this file so they're easier to spot, for example `service_name` and `service_short` could be kept together like in the other modules:

https://github.com/DFE-Digital/terraform-modules/blob/83801213853ed1e4b4bdcb8d36773c8683ff010f/aks/postgres/variables.tf#L22-L30

Reply:
changed as requested